app/src/main/java/com/aleksandra/go4mytrip/ui/main/clothes/PackingClothesContract.kt

```diff
@@ -0,0 +1,14 @@
+package com.aleksandra.go4mytrip.ui.main.clothes
+
+import com.aleksandra.go4mytrip.lists.PackingModel
+import com.google.firebase.database.DatabaseReference
+
+interface PackingClothesContract {
+
+    interface View {
+        fun showPackingList(packingList: MutableList<PackingModel>, isPacking: Boolean)
+        }
+    interface Presenter {
+       fun fetchPackingList(idTrip : String, referencePackingList: DatabaseReference, uid: String)
```

Comment:
you shouldn't reference DatabaseReference here, repository should be passed to the presenter and used internally